pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,191 @@
+      - title: EKS
+        src: /assets/img/logos-technologies/logo-eks.png
+
+  - heading: "Step 2: Deploy your EKS cluster"
+    tab_code_preview: site.data.enterprise-landing-page-tabs-code
+    image_side: left
+    body: |
+      Deploy your custom configured EKS cluster. Our friendly user interface can generate code for your existing AWS
+      accounts and VPCs, or you can dive right in to the code. Or, use <a href="https://gruntwork.io/landing-zone-for-aws/">Gruntwork Landing Zone</a> and the
+      <a href="https://gruntwork.io/reference-architecture">Gruntwork Reference Architecture</a> for a complete,
+      end-to-end stack, built with extensibility in mind. No matter what you choose, everything is managed as code with Terraform.
+    logos:
+      - title: Terraform
+        src: /assets/img/landing-heroku-page/terraform-icon.png
+      - title: Docker
+        src: /assets/img/landing-heroku-page/docker-icon.png
+      - title: Packer
+        src: /assets/img/landing-heroku-page/packer-icon.png
+
+  - heading: "Step 3: Set up your app"
+    image: /assets/img/landing-page-enterprise/sample-app-clip.gif
+    image_side: right
+    body: |
+      Use Gruntwork tools to optimize your enterprise apps for EKS. Update your app with cloud-native best practices for
+      packaging, monitoring, logging, service discovery, secrets management, schema migrations, and more. Or customize
+      the optimizations yourself using Gruntwork sample apps as a template. We include sample apps common enterprise languages and
+      frameworks:
+      <ul>
+          <li>Spring boot</li>
+          <li>Standard Java apps (e.g., Tomcat or similar)</li>
+          <li>Ruby on Rails</li>
+          <li>Python Django</li>
+          <li>Or suggest something!</li>
+      </ul>
+    logos:
+      - title: Spring Boot
+        src: /assets/img/logos-technologies/logo-spring-boot.png
+      - title: Rails
+        src: /assets/img/logos-technologies/logo-rails.png
+      - title: Django
+        src: /assets/img/logos-technologies/logo-django.png
+
+  - heading: "Step 4: Run your app"
+    tab_block: site.data.enterprise-landing-page-tabs-images
+    image_side: left
+    body: |
+      Gruntwork Enterprise EKS supports various deployment strategies out of the box, including canary, blue/green
+      deployments, automatic rollbacks, security scans, CI/CD, GitOps workflows, etc. Post deployment, use powerful
+      observability tools to manage your apps, including dashboards for key metrics and cluster data, one-click
+      access to containers, and multiple options for centralized logging. Stay up to date automatically (see Patcher<!-- TODO: link to patcher page-->).
+    logos:
+      - title: Prometheus
+        src: /assets/img/logos-technologies/logo-prometheus.png
+      - title: Datadog
+        src: /assets/img/logos-technologies/logo-datadog.png
+
+  - heading: "Step 5: Get help and support"
```

Comment:
```suggestion
  - heading: "Step 5: Stay up to date"
```

I think the key thing to highlight here is that K8S, EKS, AWS, Terraform, and Docker are constantly changing, and that Gruntwork provides commercial maintenance for the code, so you can update your EKS cluster and services to the latest best practices and security patches with a version number bump.

Reply:
Updated this section to be focused on patcher.